### PR TITLE
[ADD] pc_stock_connect_warehouse

### PR DESCRIPTION
**Pull request by ghost**
Added number of files to attach to WAB files

### CODE REVIEW

File: pc_connect_warehouse_yellowcube/account_invoice_ext.py [hunk 1]
```diff
@@ -73,29 +80,113 @@ def get_attachment_wab(self, cr, uid, ids, extension='pdf', context=None):
...
+                    attachment_content_base64 = base64.b64encode(f.read())
+                values_create_att = {
+                    'name': output_filename,
+                    'datas': attachment_content_base64,
+                    'datas_fname': output_filename,
+                    'res_model': 'account.invoice',
+                    'res_id': invoice.id,
+                    'type': 'binary',
+                    'description':
+                        _('Attachment for invoice with ID={0}. '
```
Comment (BT-cserra) [inline pc_connect_warehouse_yellowcube/account_invoice_ext.py:176]: Move the format outside the call of `_()`.

File: pc_connect_warehouse_yellowcube/stock_picking_ext.py [hunk 2]
```diff
@@ -212,11 +202,12 @@ def _get_attachment_id_for_picking_and_barcode_concatenated(self, cr, uid, ids,
                     'res_model': 'stock.picking',
                     'res_id': stock_picking_out.id,
                     'type': 'binary',
-                    'description': _('Attachment for picking with ID={0}. It is the concatenation of '
-                                     'the attachments of the picking (attach. id={1}) '
-                                     'and the barcode report (attach. ID={2}'.format(stock_picking_out.id,
-                                                                                     att_picking_id,
-                                                                                     att_barcode_id)),
+                    'description':
+                        _('Attachment for picking with ID={0}. '
```
Comment (BT-cserra) [inline pc_connect_warehouse_yellowcube/stock_picking_ext.py:206]: The format is inside the `_()` call. Should be outside.

File: pc_connect_warehouse_yellowcube/stock_picking_ext.py [hunk 3]
```diff
@@ -117,54 +117,43 @@ def get_attachment_wab(self, cr, uid, ids, extension='pdf', context=None):
...
-        # if no barcode needs to be attached, BUT it will be *a new* attachment, created ad-hoc, with is the
-        # concatenation of the picking and the barcode.
-        if not att_barcode_ids:
-            attachment_to_send_id = att_picking_ids[0]  # Just the (only) delivery slip found.
+        if len(att_picking_ids) == 1:
+            attachment_to_send_id = att_picking_ids[0]
         else:
-            # Here we need to create a new attachment which is the concatenation of the picking and the barcode.
-            attachment_to_send_id = stock_picking_out._get_attachment_id_for_picking_and_barcode_concatenated(att_picking_ids[0], att_barcode_ids[0], extension)
+            attachment_to_send_id = stock_picking_out.\
```
Comment (BT-cserra) [inline pc_connect_warehouse_yellowcube/stock_picking_ext.py:145]: All those `\` make the code difficult to read sometimes. Is it strictly required to adhere to a maximum line-length?

Comment (ghost) [reply pc_connect_warehouse_yellowcube/stock_picking_ext.py:145]: It makes easier to read on github, as very long lines mess with the web UI. Also, it helps when debugging on server

**Review by BT-cserra — APPROVED** [review]

**Pull request merged**